src/syntax/syntax.rkt

```diff
@@ -301,7 +392,9 @@
   [->tex (curry format "\\log_* \\left( \\mathsf{gamma} ~a \\right)")])
 
 (define-operator/libm (log real) real
-  [libm log logf] [bf bflog] [cost 70]
+  [libm log logf]
+  [type (hash 1 '(((real) real) ((complex) complex)))]
```

Comment:
Will this actually work? I get the sense that this will crash if you attempt to evaluate `sqrt` on a complex number, since we're just directly using the `libm` version. If it doesn't work (if running Herbie on `(sqrt (complex 1 0))` crashes) we should remove this type we allow for `log` and the similar things below.

Reply:
It will indeed not work and I have deleted disabled complex for sqrt, exp and log. sqr works though so I left complex legal for sqr